### PR TITLE
Different way of stopping overscroll

### PR DESCRIPTION
**Pull request by aghassemi**
First attempt to fix the Safari bug in viewer where scrolling carousel sometimes scrolls the viewer.

### CODE REVIEW

**Comment by sriramkrish85** [comment]
may be add a few lines in the PR explaining how it prevents the issue.

**Comment by sriramkrish85** [comment]
(Also please test on all devices)

File: extensions/amp-carousel/0.1/amp-carousel.css [hunk 1]
```diff
@@ -103,7 +103,7 @@ amp-carousel .amp-carousel-button.amp-disabled {
 }
 
 .-amp-slides-container.-amp-no-scroll {
-  overflow-x: hidden !important;
+  -webkit-overflow-scrolling: auto !important;
```
Comment (jridgewell) [inline extensions/amp-carousel/0.1/amp-carousel.css]: This won't stop scrolling on non-iOS devices.

Comment (aghassemi) [inline extensions/amp-carousel/0.1/amp-carousel.css]: Not needed, only needed where there is momentum scrolling. Have tested with firefox and is fine.

**Comment by aghassemi** [comment]
I still don't have a viewer setup, I'm gonna test this more before merging.

**Comment by aghassemi** [comment]
never mind, the issue it's deeper than this :) second attempt PR (very different fix) to follow

**Pull request closed without merging**